Tools/Frame_params/Solo_Copter-3.6_GreenCube.param

```diff
@@ -41,9 +41,9 @@ BATT_MONITOR,5
 BRD_SAFETYENABLE,0
 BRD_SAFETY_MASK,16368
 CIRCLE_RADIUS,3000
-COMPASS_ORIENT,38
+COMPASS1_ORIENT,38
```

Comment:
little bit of overreach here given these are for specific AC versions

Reply:
there are a few instances of this

Reply:
yep, fixed.